### ISSUE TITLE
Script for retrieving static field addresses is invalid

### ISSUE THREAD

**Issue by Symbai** [cheat-engine/cheat-engine#2841]
il2cpp, CE's generated script to retrieve static field address is invalid:

![image](https://github.com/cheat-engine/cheat-engine/assets/14368203/4dc14004-c839-4ff6-b2c3-04383da32045)

**Comment by cheat-engine** [comment]
yup, it shouldn't even be an option to generate that script on il2cpp
